plugins/node/opentelemetry-plugin-dns/test/integrations/dnspromise-lookup.test.ts

```diff
@@ -122,52 +132,77 @@ describe('dns.promises.lookup()', () => {
     it('should export a valid span with error INVALID_ARGUMENT when "family" param is equal to -1', async () => {
       const hostname = 'google.com';
       try {
-        await dns.promises.lookup(hostname, -1);
+        await lookupPromise(hostname, { family: -1 });
         assert.fail();
       } catch (error) {
         const spans = memoryExporter.getFinishedSpans();
         const [span] = spans;
 
         assert.strictEqual(spans.length, 1);
-        assertSpan(span, {
-          addresses: [],
-          hostname,
-          forceStatus: {
-            code: CanonicalCode.INVALID_ARGUMENT,
-            message: error!.message,
-          },
-        });
+        if (semver.lt(process.versions.node, '9.0.0')) {
```

Comment:
I created a new dns lookup method in this file that is compatible with node 8. The only thing is that the node `INVALID_ARGUMENT` code seemed to be different in node 8 so I had to make separate cases for this test. If this is fine then I'll open this PR to be merged.

Reply:
Any idea why the canonical code is different between them? I would rather have behavior consistent across versions if possible

Reply:
base on [this](https://nodejs.org/api/errors.html#errors_error_code) it seems like it's possible to have different error codes in different major releases. The error message was the same though.